Documentation/how-to-build-WebAssembly.md

```diff
@@ -32,8 +32,9 @@ This is Windows only for now.
 2. To compile that to WebAssembly, run ```emcc HelloWasm.bc  -s ALLOW_MEMORY_GROWTH=1  C:\corert\bin\WebAssembly.wasm.Debug\sdk\libPortableRuntime.bc C:\corert\bin\WebAssembly.wasm.Debug\sdk\libbootstrappercpp.bc -s WASM=1 -o HelloWasm.html``` (if emcc isn't on your path, you'll need to launch an Emscripten command prompt to do this). That will generate a .wasm file with your code as well as html and js files to run it.
 
 # How to run a WebAssembly application #
-1. Ensure you have Edge 41 (Windows only) or above or [Firefox](https://www.getfirefox.com).
-2. Open the generated html file in Edge or Firefox and look at the on-screen console for output.
+1. Ensure you have Edge (Windows only), Chrome or [Firefox](https://www.getfirefox.com).
+2. Launch HTTP server which will serve folder with generated html/wasm files. For example `http-server` from npm will serve, but any other option can do too. Without HTTP server, browser will prevent donwloading wasm file.
```

Comment:
This is good, but its an option.  You dont have to use an HTTP server,  FIrefox can launch the html/wasm without one, although you'll need to change the CORS setting for local files:

![image](https://user-images.githubusercontent.com/2720041/75198146-e4ae6280-572d-11ea-8039-4e4ef9b3f951.png)

and 
https://support.mozilla.org/en-US/questions/1264280
Maybe you can add that info?

Reply:
@yowl thanks for suggestion, it is much more convenient when playing with WASM builds. I update docs